File: tl/src/main/java/com/github/badoualy/telegram/tl/api/TLNotifyChats.java
package com.github.badoualy.telegram.tl.api;

/**
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
public class TLNotifyChats extends TLAbsNotifyPeer {
    public static final int CONSTRUCTOR_ID = 0xc007cec3;

    public TLNotifyChats() {
    }

    @Override
    public String toString() {
        return "notifyChats#c007cec3";
    }

    @Override
    public int getConstructorId() {
        return CONSTRUCTOR_ID;
    }
}

File: tl/src/main/kotlin/com/github/badoualy/telegram/tl/api/TLNotifyChats.kt
package com.github.badoualy.telegram.tl.api

/**
 * notifyChats#c007cec3
 *
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
class TLNotifyChats : TLAbsNotifyPeer() {
    private val _constructor: String = "notifyChats#c007cec3"

    override val constructorId: Int = CONSTRUCTOR_ID

    override fun toString() = _constructor

    override fun equals(other: Any?): Boolean {
        if (other !is TLNotifyChats) return false
        if (other === this) return true

        return true
    }
    companion object  {
        const val CONSTRUCTOR_ID: Int = 0xc007cec3.toInt()
    }
}
